releases.json

```diff
@@ -42,7 +42,8 @@
       "argparse"
     ],
     "versions": [
-      "2.2-1"
+      "2.2-1",
+      "2.3-1"
```

Comment:
The order of these needs to be reversed.

Reply:
Thanks!
And hi :)

Reply:
hi :)